Mathlib/Analysis/Complex/UpperHalfPlane/FunctionsBoundedAtInfty.lean

```diff
@@ -85,26 +78,9 @@ theorem bounded_mem (f : ℍ → ℂ) :
 #align upper_half_plane.bounded_mem UpperHalfPlane.bounded_mem
 
 theorem zero_at_im_infty (f : ℍ → ℂ) :
-    IsZeroAtImInfty f ↔ ∀ ε : ℝ, 0 < ε → ∃ A : ℝ, ∀ z : ℍ, A ≤ im z → abs (f z) ≤ ε := by
-  rw [IsZeroAtImInfty, ZeroAtFilter, tendsto_iff_forall_eventually_mem]
-  constructor
-  · simp_rw [Filter.Eventually, atImInfty_mem]
-    intro h ε hε
-    simpa using h (Metric.closedBall (0 : ℂ) ε) (Metric.closedBall_mem_nhds (0 : ℂ) hε)
-  · simp_rw [Metric.mem_nhds_iff]
-    intro h s hs
-    simp_rw [Filter.Eventually, atImInfty_mem]
-    obtain ⟨ε, h1, h2⟩ := hs
-    have h11 : 0 < ε / 2 := by linarith
-    obtain ⟨A, hA⟩ := h (ε / 2) h11
-    use A
-    intro z hz
-    have hzs : f z ∈ s := by
-      apply h2
-      simp only [mem_ball_zero_iff, norm_eq_abs]
-      apply lt_of_le_of_lt (hA z hz)
-      linarith
-    apply hzs
+    IsZeroAtImInfty f ↔ ∀ ε : ℝ, 0 < ε → ∃ A : ℝ, ∀ z : ℍ, A ≤ im z → abs (f z) ≤ ε :=
```

Comment:
How about:
```suggestion
    IsZeroAtImInfty f ↔ ∀ ε > 0, ∃ A : ℝ, ∀ z : ℍ, A ≤ im z → abs (f z) ≤ ε :=
```

Reply:
I would prefer not to change the statement now (even up to a reducible definition).